sass/components/_channel-switcher.scss

```
@@ -94,12 +94,17 @@
             position: absolute;
             width: calc(100% - 48px);
             margin: -28px 0 0 24px;
-            background: rgba(var(--center-channel-bg-rgb), 1);
+            background: #fff;
```

We should be using the System Console theme variables here.
```suggestion
            background: rgba(var(--sys-center-channel-bg-rgb), 1);
```